jobs/component_jobs.groovy

```diff
@@ -87,15 +98,22 @@ repos.each { Map repo ->
       }
 
       publishers {
-        def statuses = [['SUCCESS', 'pending'],['FAILURE', 'failure'],['ABORTED', 'error']]
-        postBuildScripts {
-          steps {
-            statuses.each { buildStatus, commitStatus ->
-              conditionalSteps {
-                condition {
-                  status(buildStatus, buildStatus)
-                  steps {
-                    shell curlStatus(buildStatus: buildStatus, commitStatus: commitStatus, jobName: name, repoName: repo.name)
+        if (isPR) {
+          def statuses = [['SUCCESS', 'pending'],['FAILURE', 'failure'],['ABORTED', 'error']]
+          postBuildScripts {
+            onlyIfBuildSucceeds(false)
+            steps {
+              statuses.each { buildStatus, commitStatus ->
+                conditionalSteps {
+                  condition {
+                    status(buildStatus, buildStatus)
+                    steps {
+                      shell(
+                        new GroovyShell().parse(new File("${WORKSPACE}/common.groovy")).with {
```

Comment:
Was hitting errors attempting to invoke a method defined in a diff groovy script (`common.groovy` in this case).  Turns out that is a groovy thang.  http://stackoverflow.com/questions/9004303/load-script-from-groovy-script